content/en/docs/v2.3/docker_guide.md

```diff
@@ -12,13 +12,17 @@ In order to expose the etcd API to clients outside of the Docker host you'll nee
 export HostIP="192.168.12.50"
 ```
 
-The following `docker run` command will expose the etcd client API over ports 4001 and 2379, and expose the peer port over 2380.
-
-This will run the latest release version of etcd. You can specify version if needed (e.g. `quay.io/coreos/etcd:v2.2.0`).
+The following `docker run` command will expose the etcd client API over ports
```

Comment:
Docs for unsupported versions (i.e. versions < `v3.4`)  are frozen as we no longer maintain these versions.  Can you please tweak the pr so it only updates v3.4, v3.5 and v3.6? Thanks!

Reply:
Done!